modules/settings/assets/js/components/bottom-bar/index.js

```diff
@@ -1,8 +1,35 @@
 import Box from '@elementor/ui/Box';
 import Button from '@elementor/ui/Button';
+import { useSettings, useStorage, useToastNotification } from '@ea11y/hooks';
 import { __ } from '@wordpress/i18n';
 
-export const BottomBar = () => {
+const BottomBar = () => {
+	const { selectedMenu, iconDesign, iconPosition, hasChanges, setHasChanges } =
+		useSettings();
+	const { save } = useStorage();
+	const { success, error } = useToastNotification();
+
+	const saveSettings = async () => {
+		if (
+			selectedMenu.parent === 'widget' &&
+			selectedMenu.child === 'iconSettings'
+		) {
+			try {
+				await save({
+					a11y_widget_icon_settings: {
+						style: iconDesign,
+						position: iconPosition,
+					},
+				}).then(() => {
+					success('Settings saved!');
+					setHasChanges(false);
+				});
```

Comment:
You dont need to use then after await just use 
```suggestion
        await save({
					a11y_widget_icon_settings: {
						style: iconDesign,
						position: iconPosition,
					},
				});
	success('Settings saved!');
	setHasChanges(false);
```